lang-python/merge3/spec

```diff
@@ -0,0 +1,4 @@
+VER=0.0.15
+SRCS="tbl::https://github.com/breezy-team/merge3/archive/refs/tags/v$VER.tar.gz"
```

Comment:
Please use a Git tag, we forbid the use of platform-generated tarballs as the checksums may change.

https://wiki.aosc.io/zh/developer/packaging/package-styling-manual/#yuan-ma-wen-jian